x/programs/host/imports.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package host
+
+import (
+	"github.com/bytecodealliance/wasmtime-go/v14"
+
+	"github.com/ava-labs/hypersdk/x/programs/engine"
+)
+
+const (
+	wasiPreview1ModName = "wasi_snapshot_preview1"
+)
+
+var NoSupportedImports = make(SupportedImports)
+
+type Imports map[string]Import
```

Comment:
don't see where we use this type

Reply:
yeah I dropped it as it was already explicitly defined below